.github/workflows/ci.yml

```diff
@@ -74,7 +74,7 @@ jobs:
           cp tools/install/Makefile.skel Makefile
           ./build/build_atchem2.sh ./mcm/mechanism_test.fac
           if [ $RUNNER_OS != 'Linux' ] ; then
-            install_name_tool -change libopenlibm.dylib.1 $PWD/openlibm-0.4.1/libopenlibm.dylib.1 atchem2 ;
+            install_name_tool -change libopenlibm.dylib.1 $PWD/openlibm-0.8.1/libopenlibm.dylib.1 atchem2 ;
```

Comment:
Try changing to 
```
install_name_tool -change libopenlibm.dylib.1 $PWD/openlibm-0.8.1/libopenlibm.4.dylib atchem2 ;
```
(I'm not certain this is the right fix, I'll have another stab if it's wrong)

Reply:
If not, try 
```
install_name_tool -change libopenlibm.4.dylib $PWD/openlibm-0.8.1/libopenlibm.4.dylib atchem2 ;
```

Reply:
Great thanks!